src/core/RecyclerListView.tsx

```diff
@@ -481,29 +486,38 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
             }
             return (
                 <ViewRenderer key={key} data={data}
-                    dataHasChanged={this._dataHasChanged}
-                    x={itemRect.x}
-                    y={itemRect.y}
-                    layoutType={type}
-                    index={dataIndex}
-                    styleOverrides={styleOverrides}
-                    layoutProvider={this.props.layoutProvider}
-                    forceNonDeterministicRendering={this.props.forceNonDeterministicRendering}
-                    isHorizontal={this.props.isHorizontal}
-                    onSizeChanged={this._onViewContainerSizeChange}
-                    childRenderer={this.props.rowRenderer}
-                    height={itemRect.height}
-                    width={itemRect.width}
-                    itemAnimator={Default.value<ItemAnimator>(this.props.itemAnimator, this._defaultItemAnimator)}
-                    extendedState={this.props.extendedState} />
+                              dataHasChanged={this._dataHasChanged}
+                              x={itemRect.x}
+                              y={itemRect.y}
+                              layoutType={type}
+                              index={dataIndex}
+                              styleOverrides={styleOverrides}
+                              layoutProvider={this.props.layoutProvider}
+                              forceNonDeterministicRendering={this.props.forceNonDeterministicRendering}
+                              isHorizontal={this.props.isHorizontal}
+                              onSizeChanged={this._onViewContainerSizeChange}
+                              childRenderer={this.props.rowRenderer}
+                              height={itemRect.height}
+                              width={itemRect.width}
+                              itemAnimator={Default.value<ItemAnimator>(this.props.itemAnimator, this._defaultItemAnimator)}
+                              extendedState={this.props.extendedState}/>
             );
         }
         return null;
     }
 
     private _onViewContainerSizeChange = (dim: Dimension, index: number): void => {
         //Cannot be null here
-        (this._virtualRenderer.getLayoutManager() as LayoutManager).overrideLayout(index, dim);
+        const layoutManager: LayoutManager = this._virtualRenderer.getLayoutManager() as LayoutManager;
+
+        if (this.props.debugHandler
+            && this.props.debugHandler.resizeDebug
+            && (__DEV__ || this.props.debugHandler.getDebugConfig().forceEnable)) {
```

Comment:
If we rely on DEV it would mean that code bases will not handle removing this config on prod builds. __DEV__ is not set on web.

Reply:
Lets always take from client who is integrating debugging.

Reply:
will remove the check for production.